### PR TITLE
UI changes in basic info modal

### PR DESCRIPTION
**Pull request by evulins**
Trello cards:
[Calendar button](https://trello.com/c/VRX5jcSb/1824-session-details-calendar-view)
[Display modal](https://trello.com/c/1SKiUULj/1823-session-details-basic-data)
[Marker zoom](https://trello.com/c/me7xXEFB/1893-map-close-session-graph-view-map-view-should-return-to-the-view-prior-to-the-selection-of-the-session)

### CODE REVIEW

**Review by smialko — COMMENTED** [review]
Tested - The X button has blue border when I enter modal on desktop. On mobile it doesn't. See the photo below: 
Screenshot 2024-06-05 at 12 34 07

**Comment by evulins** [comment]
> Tested - The X button has blue border when I enter modal on desktop. On mobile it doesn't. See the photo below: Screenshot 2024-06-05 at 12 34 07

I believe Janek is doing global changes for this, but sure i can fix it also here :)

File: app/javascript/react/components/Modals/SessionDetailsModal/SessionDetailsModal.style.tsx [hunk 1]
```diff
@@ -155,6 +155,10 @@ const CancelButtonX = styled.button`
   border: none;
   background-color: transparent;
   width: fit-content;
+
+  &:focus-visible {
```
Comment (jankrzempek) [inline app/javascript/react/components/Modals/SessionDetailsModal/SessionDetailsModal.style.tsx:159]: DO NOT TAKE MY WORK (I already have less things) 😜

Comment (smialko) [reply app/javascript/react/components/Modals/SessionDetailsModal/SessionDetailsModal.style.tsx:159]: 😂

**Review by smialko — APPROVED** [review]
👏

**Pull request merged**